internal/configuration/settings/portforward.go

```diff
@@ -29,6 +29,11 @@ type PortForwarding struct {
 	// to write to a file. It cannot be nil for the
 	// internal state
 	Filepath *string `json:"status_file_path"`
+	// Command is the port forwarding status command
```

Comment:
Self comment to self-resolve: don't forget to merge PR https://github.com/qdm12/gluetun-wiki/pull/89 !